README.md

```diff
@@ -0,0 +1,118 @@
+# Sprinter TS - Cross-Chain Developer Toolkit
+
+## Table of Contents
+
+- [Sprinter TS - Cross-Chain Developer Toolkit](#sprinter-ts---cross-chain-developer-toolkit)
+    - [Introduction](#introduction)
+    - [Prerequisites](#prerequisites)
+    - [Getting Started](#getting-started)
+    - [Building](#building)
+    - [Testing](#testing)
+    - [Linting and Formatting](#linting-and-formatting)
+
+---
+
+<a name="introduction"></a>
+## Introduction
+
+Sprinter TS is a powerful developer toolkit designed to simplify cross-chain integrations and enhance the developer experience. It includes:
+
+- **TypeScript SDK** for strongly typed APIs and simplified interactions.
+- **React SDK** with hooks and context for seamless integration into React applications.
+- **Comprehensive Documentation** to guide and support developers.
+
+Whether you're building cross-chain applications or experimenting with decentralized technologies, Sprinter provides the tools you need to succeed.
+
+---
+
+<a name="prerequisites"></a>
```

Comment:
When I tested in the rendered preview, the links above didn't bring to the titles. I guess having an `id` instead would work better. cf: https://stackoverflow.com/questions/484719/should-i-make-html-anchors-with-name-or-id
```suggestion
<a id="prerequisites"/>
```

Reply:
Hmm I see the opposite in this link indeed.. just it didn't work in the preview for me :shrug: 
https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown